src/stravalib/protocol.py

```diff
@@ -166,8 +166,8 @@ def exchange_code_for_token(
         client_id: int,
         client_secret: str,
         code: str,
-        return_athlete: bool | None = False,
-    ) -> AccessInfo | tuple[AccessInfo, dict[str, Any]]:
+        return_athlete: bool = False,
+    ) -> tuple[AccessInfo, dict[str, Any] | None]:
```

Comment:
The typing issue might have been caused by some ambiguity in the previous version's signature. The typing issue was resolved by making `protocol.exchange_code_for_token()` always return a tuple. The athlete part of the returned tuple will be None if not requested or not present in Strava's response.